webapp/src/headerbar.tsx

```diff
@@ -239,7 +239,7 @@ export class HeaderBar extends data.Component<ISettingsProps, {}> {
         const { home, header, tutorialOptions } = this.props.parent.state;
         const isController = pxt.shell.isControllerMode();
         const isNativeHost = cmds.isNativeHost();
-        const hasIdentity = auth.hasIdentity();
+        const hasIdentity = auth.hasIdentity() && !pxt.auth.proxyIdentityThroughIPC();
```

Comment:
The webapp calls auth.hasIdentity() in a lot of places. Should this check be done at a lower level so the behavior is consistent?

Reply:
all this is doing is turning off the login/profile button in the header bar. we want all of the other cloud ui/behavior to be the same as usual, but there's no login/profile because this isn't a makecode account